lib/src/html_input_stream.dart

```diff
@@ -292,3 +292,26 @@ String codecName(String encoding) {
   var canonicalName = encoding.replaceAll(asciiPunctuation, '').toLowerCase();
   return encodings[canonicalName];
 }
+
```

Comment:
Can you copy over the existing dartdoc comments for these functions as well?

Reply:
yup!